tests/staticfiles_tests/project/documents/cached/module.js

```diff
@@ -20,3 +20,4 @@ export {
     firstVar as firstVarAlias,
     secondVar as secondVarAlias
 } from "./module_test.js";
+//# sourceMappingURL=data:application/json;charset=utf8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIl9zcmMv
```

Comment:
Please move it to the `source_map.js`.

Reply:
Made it a separate test named test_js_source_map_uri with a source_map_data_uri.js and the same for CSS.